_doc-build-linux/action.yml

```
@@ -250,6 +306,8 @@ runs:
       if: inputs.requires-xvfb == 'true'
       shell: bash
       run: |
+        source .venv/bin/activate
+        cd doc
         xvfb-run ${{ env.SPHINX_BUILD_MAKE }} -C doc html SPHINXOPTS="${{ inputs.sphinxopts }}"
         xvfb-run ${{ env.SPHINX_BUILD_MAKE }} -C doc pdf
```

```suggestion
        xvfb-run ${{ env.SPHINX_BUILD_MAKE }} -C doc html SPHINXOPTS="${{ inputs.sphinxopts }}"
        xvfb-run ${{ env.SPHINX_BUILD_MAKE }} -C doc pdf
```
If you are navigating to the "doc" folder you don't need ``-C doc``. Or viceversa. I took the second approach (not navigating) but up to you :)